sound/soc/sof/ipc4.c

```diff
@@ -545,6 +545,8 @@ static int ipc4_fw_ready(struct snd_sof_dev *sdev, struct sof_ipc4_msg *ipc4_msg
 	sdev->host_box.offset = outbox_offset;
 	sdev->host_box.size = outbox_size;
 
+	sdev->debug_box.offset = snd_sof_dsp_get_window_offset(sdev, 2);
```

Comment:
Add a define/constant for the magic value?

Reply:
I'll use `SOF_IPC4_DEBUG_WINDOW_IDX` and add `SOF_IPC4_OUTBOX_WINDOW_IDX` for the other window we are asking for with index.